lib/Backends/CPU/tests/CPUOperatorTest.cpp

```diff
@@ -26,6 +26,8 @@ std::set<std::string> glow::backendTestBlacklist = {
     "Logit_Float16/0",
     "FP16Add/0",
     "FP16Matmul/0",
+    "BroadCastMax/0",
```

Comment:
Why is it black listed from CPU?

Reply:
Looks like `InsertTensor` doesn't have support for `Int64ITy` in the CPU or in OpenCL.

Reply:
Oh. I thought type dimension demotion 64 --> 32 would kick in.
I guess it's besides the point since CPU backend doesn't support int32 or 64 anyway.
return NI.allInputsAndOutputsHaveSameElemKind(
        {ElemKind::FloatTy, ElemKind::Int8QTy});